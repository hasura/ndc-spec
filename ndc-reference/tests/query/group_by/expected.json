[
  {
    "groups": [
      {
        "dimensions": {
          "author": 1
        },
        "aggregates": {
          "min_id": 1,
          "max_id": 1
        },
        "rows": [
          {
            "id": 1,
            "title": "The Next 700 Programming Languages"
          }
        ]
      },
      {
        "dimensions": {
          "author": 2
        },
        "aggregates": {
          "min_id": 2,
          "max_id": 3
        },
        "rows": [
          {
            "id": 2,
            "title": "Why Functional Programming Matters"
          },
          {
            "id": 3,
            "title": "The Design And Implementation Of Programming Languages"
          }
        ]
      }
    ]
  }
]