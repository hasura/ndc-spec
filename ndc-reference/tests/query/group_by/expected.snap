---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/group_by/request.json
---
[
  {
    "groups": [
      {
        "dimensions": [
          1
        ],
        "aggregates": {
          "min_id": 1,
          "max_id": 1
        }
      },
      {
        "dimensions": [
          2
        ],
        "aggregates": {
          "min_id": 2,
          "max_id": 3
        }
      }
    ]
  }
]
